requirements/edx/github.txt

```diff
@@ -21,7 +21,7 @@
 git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a0c695#egg=django-cas
 
 # Our libraries:
--e git+https://github.com/edx/XBlock.git@9c634481dfc85a17dcb3351ca232d7098a38e10e#egg=XBlock
+-e git+https://github.com/edx-solutions/XBlock.git@726847de5b2fba3c666722c73e32bac73ad6e794#egg=XBlock
```

Comment:
See https://github.com/edx/XBlock/pull/286 - We'll want to update this when in place.